docs/operator-cli-guide.md

```diff
@@ -0,0 +1,88 @@
+## Operator CLI
+
+The `wcn_operator` application allows node operators to manage their nodes. The application can be run from the repository via:
```

Comment:
Not just nodes, but rather their smart-contract state - representation of their infra in the SC.